src/EventStore.ClusterNode/Program.cs

```diff
@@ -167,8 +167,40 @@ internal static class Program {
 					Application.Exit(0, "Cancelled.");
 				};
 
-				var metricsConfiguration = MetricsConfiguration.FromFile();
-				using (var hostedService = new ClusterVNodeHostedService(options, certificateProvider, metricsConfiguration.Get<MetricsConfiguration>())) {
+				// Create a single IConfiguration object that contains the whole configuration, including 
+				// plugin configuration. We will add it to the DI and make it available to the plugins.
+				//
+				// Three json files are loaded explicitly for backwards compatibility
+				// - metricsconfig.json needs loading into the EventStore:Metrics section.
+				// - kestrelsettings.json is not located in a config/ directory
+				// - logconfig.json is not located in a config/ directory
+				var configuration = new ConfigurationBuilder()
+					.AddSection(SectionNames.EventStore, eventStoreSection => eventStoreSection
```

Comment:
could we simply use `EventStore:Metrics`  as the section name?

`.AddSection("EventStore:Metrics", x => x.AddConfigFile("metricsconfig.json"))`